lib/definitions/project.d.ts

```diff
@@ -20,6 +20,10 @@ interface IProjectTemplatesService {
 	defaultTemplatePath: IFuture<string>;
 }
 
+interface IPlatformProjectServiceBase {
+	getPluginPlatformsFolderPath(pluginData: IPluginData, platform: string): string;
```

Comment:
Some JSDocumentation would be nice for this interface and the `addLibrary` function below